libs/journeys/ui/src/components/TemplateView/CreateJourneyButton/CreateJourneyButton.tsx

```diff
@@ -13,6 +13,12 @@ import { AccountCheckDialog } from '../AccountCheckDialog'
 
 interface CreateJourneyButtonProps {
   signedIn?: boolean
+  /* If true, the team dialog will be opened when the button is clicked.
+   * If you have multiple instances of this button on a page,
+   * set this to false for all but the first button
+   * so that you don't get multiple team dialogs displayed after logging in.
+   */
```

Comment:
```suggestion
   */
```
Update this comment to make it clearer:
/**
 * Whether to automatically open the team dialog when the user signs in
 * with createNew=true in the URL. Only one button per page should have
 * this set to true to avoid multiple dialogs opening.
 */